requirements.txt

```diff
@@ -11,6 +11,6 @@ google-auth-oauthlib==1.1.0
 googleapis-common-protos==1.61.0
 opensearch-py==2.4.1
 pandas==2.1.3
-sqlalchemy==1.4.44
+sqlalchemy==2.0.23
```

Comment:
https://docs.sqlalchemy.org/en/20/changelog/migration_20.html

Should spend some time looking into this before merging